### PR TITLE
[develop] Mauricio - Create card test

### PR DESCRIPTION
**Pull request by Snip-77**

### CODE REVIEW

File: config.json.dist [hunk 1]
```diff
@@ -6,14 +6,14 @@
     "accounts": [
       {
         "user": "user1",
-        "username": "",
+        "email": "",
         "password": "",
```
Comment (luqp) [inline config.json.dist:11]: Username is required for API to remove items created by UI, it should not be removed

File: src/main/java/org/fundacionjala/trello/pages/card/CardPage.java [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+
+import java.util.ArrayList;
+import java.util.List;
+
+
```
Comment (luqp) [inline src/main/java/org/fundacionjala/trello/pages/card/CardPage.java]: There is an unnecessary  white space.

File: src/main/java/org/fundacionjala/trello/pages/card/CardPage.java [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+     * @return true if actual page is displayed, else false.
+     */
+    @Override
+    public boolean isDisplayed() {
+        return true;
+    }
+
+    public List<String> getAllCardNames(final String listName) {
+        String getCardsList = String.format(CARD_NAMES_LIST, listName);
+        List<WebElement> test = driver.findElements(By.cssSelector(ALL_CARDS_LIST));
```
Comment (luqp) [inline src/main/java/org/fundacionjala/trello/pages/card/CardPage.java]: You can also get this List by defining it as `@FindBy`

File: src/test/java/org/fundacionjala/trello/TDD/card/CreateCardTest.java [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+        list.put("idBoard", "{board.id}");
+        response = requestManager.init(context).queryParams(list).post(LIST.getEndPoint());
+        context.saveResponse(LIST.name().toLowerCase(), response);
+        context.saveIds(LIST, response.jsonPath().getString("id"));
+    }
+
+    @AfterMethod
+    public void deleteBoard() {
+        context.getIdsByKey(BOARD)
+                .forEach(id -> requestManager.init(context).delete("/boards/".concat(id)));
```
Comment (luqp) [inline src/test/java/org/fundacionjala/trello/TDD/card/CreateCardTest.java:47]: If you're not going to delete lists, why are you saving them in context?

**Review by luqp — COMMENTED** [review]
Good job, I leave some comments, please take a look.

File: config.json.dist (hunk 1 above)
Comment (Snip-77) [reply config.json.dist:11]: Ok, ill put both then, is that ok?

File: src/test/java/org/fundacionjala/trello/TDD/card/CreateCardTest.java (hunk 4 above)
Comment (Snip-77) [reply src/test/java/org/fundacionjala/trello/TDD/card/CreateCardTest.java:48]: Good observation, working on it

File: src/main/java/org/fundacionjala/trello/pages/card/CardPage.java (hunk 3 above)
Comment (Snip-77) [reply src/main/java/org/fundacionjala/trello/pages/card/CardPage.java]: Good observation, working on it...

File: src/test/java/org/fundacionjala/trello/TDD/card/CreateCardTest.java (hunk 4 above)
Comment (Snip-77) [reply src/test/java/org/fundacionjala/trello/TDD/card/CreateCardTest.java:47]: Good point, done

**Review by carlosmc23 — APPROVED** [review]
Good job Mauricio

**Pull request merged**